overlay/master_server.py

```diff
@@ -648,7 +641,7 @@ def run(master_port=50051) -> None:
 
     # Initialize server and master
     master_hostname: str = socket.gethostname()
-    server = grpc.server(ThreadPoolExecutor(max_workers=10))
+    server = grpc.server(ThreadPoolExecutor(max_workers=32))
```

Comment:
Why the increment?

Reply:
I'm not sure! I believe Caleb made this change.